Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM yastdevel/ruby:sle15
+
```

Comment:
np: why this blank line?

Reply:
I found it more helpful and it was in SP4, so for easier future merges. But in general not needed. Just when I see it in conflict I like it more with empty line.